README.md

```diff
@@ -1 +1,5 @@
-# review-project
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
- [ ] [REQUIRED] Well done on phrasing your comment politely, however, you still need to include an explanation, a "why". It will be nice if you let @Tufahel know why the README needs to be more descriptive.